mix.exs

```diff
@@ -113,7 +114,7 @@ defmodule Orbit.MixProject do
       test_all: [
         "test",
         "credo",
-        "format --check-formatted",
+        "Format --check-formatted",
```

Comment:
**question (low): ** what was this for?

Reply:
Ugh, sorry, that was an error that I thought I removed in a fixup commit. Should be actually fixed in c9baca6.